src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultLSPEditorFeatureDetector.cs

```diff
@@ -160,12 +161,12 @@ private protected virtual bool EnvironmentFeatureEnabled()
         // Private protected virtual for testing
         private protected virtual bool IsFeatureFlagEnabled()
         {
-            if (_featureFlags.IsFeatureEnabled(RazorLSPEditorFeatureFlag, defaultValue: false))
-            {
-                return true;
+            if (!_featureFlagEnabled.HasValue)

```

Comment:
Would you mind also updating the [text](https://github.com/dotnet/aspnetcore-tooling/blob/8edd1c4cd5d25a4229b206bfa6e665095754bb69/src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef#L15) in our pkgdef to state that a VS restart is required?

Reply:
Sure, will make a change to the text. Let me know if you were thinking something different.

Reply:
Looks perfect!